CMakeLists.txt

```diff
@@ -1217,6 +1217,29 @@ else ()
   set(HAVE_HALF_FLOAT OFF)
 endif ()
 
+# Define a macro to check if we have included and initialized libipt.
+set(HAVE_LIBIPT OFF)
```

Comment:
This is a submodule and we have documented that it should always be updated and added it to the developer workflow commands, so do we need this variable?

Reply:
Maybe the variable should remain but should only be set for LINUX AND X64 so that arch constraint doesn't have to be repeated everywhere like in suite/tests/.

Reply:
Done.